kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/HighAvailabilityConf.scala

```diff
@@ -146,5 +146,5 @@ object HighAvailabilityConf {
       .doc("Whether we should publish Kerberos configs to ZooKeeper.")
```

Comment:
Can we add some description of HIVE-11581? e.g. the minimal compatible version of hive client